### ISSUE TITLE
Error using PowerShell

### ISSUE THREAD

**Issue by MikeTheLiberal** [PowerShell/PSReadLine#1830]
I receive errors when trying to run the Docker tutorial.

-----------------------------------------------------------------------
Last 124 Keys:
 c d Space g e t t i n g - s t a r t e d Space Enter       
 Space d o c k e r Space b u i l d Space - t Space d o c k 
e r 1 0 1 t u t o r i a l Space . Space Enter
 d o c k e r Space r u n Space - d Space - p Space 8 0 : 8 
0 Space - - n a m e Space d o c k e r - t u t o r i a l Spa
ce d o c k e r 1 0 1 t u t o r i a l Enter
 d o

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender() 
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()   
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------

**Comment by antondubinkin** [comment]
same issue
prerequisites
server 2012 r2, RDP to the server, PS7 was copied to the desktop of the server as a standalone copy taken from a machine where it was previously installaed via downloaded installation. the issue is reproduciable constantly. 
Steps: type gci c:\ -(then type ctrl+space) and immediatelly the issue pops up as on the screenshot. 
PS C:\Users\username\Desktop\7> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 6.3.9600
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0.}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
 
### Exception
```
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension. (Parameter 'top')
Actual value was 300.
   at System.ConsolePal.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.Menu.UpdateMenuSelection(Int32 selectedItem, Boolean select, Boolean showTooltips, String toolTipColor)
   at Microsoft.PowerShell.PSConsoleReadLine.MenuCompleteImpl(Menu menu, CommandCompletion completions)
   at Microsoft.PowerShell.PSConsoleReadLine.PossibleCompletionsImpl(CommandCompletion completions, Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.CompleteImpl(Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.MenuComplete(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(PSKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```


![2020-09-24_00h52_59](https://user-images.githubusercontent.com/29511875/94073525-4fc53200-fe00-11ea-9350-e0cee160dd1b.png)

**Comment by daxian-dbw** [comment]
@MikeTheLiberal This issue you reported was fixed (see #1306). Please upgrade to the 2.0.4 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

@antondubinkin The issue you reported here was a different one, and I moved to a separate new issue: https://github.com/PowerShell/PSReadLine/issues/1851.
